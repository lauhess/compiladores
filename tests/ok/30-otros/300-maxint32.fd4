let (maxInt32:Nat)  = print "max int 32: " 2147483647
let (maxUInt32:Nat) = print "max unsigned int 32: " 4294967295 
