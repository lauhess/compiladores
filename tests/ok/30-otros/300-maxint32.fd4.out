max int 32: 2147483647
max unsigned int 32: 4294967295
